test/channel/base_test.rb

```diff
@@ -92,6 +111,23 @@ def rm_rf
     assert_nil @channel.last_action
   end
 
+  test "try calling a public method defined on Base" do
+    @channel.perform_action 'action' => :kick
+    assert_nil @channel.last_action
```

Comment:
I've renamed the tests related to action methods to be more clear. 
I belive we still have and issue more related to the ways channels work as when the client sends an forbidden action the channel will log the request and do nothing. I think we might want to respond to the client with a forbidden action message.